release_tools/semverup.py

```diff
@@ -66,11 +72,14 @@ def semverup(dry_run):
     Additionally, 'pyproject' file will also be updated. Take into
     account this file must be tracked by the repository.
 
-    WARNING: this script does not increases MAJOR version yet.
+    WARNING: this script does not increase MAJOR version yet.
```

Comment:
We should add something like "unless it is forced using..." the name of the option to do it.